clirr-ignored-differences.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
don't add methods to interfaces without a major version bump

Reply:
@elharo As of now,every handwritten client follow the same behaviour if there is a new feature comes and that requireds api call,then for the rpc call we are adding it in interface,without adding methods in interface it wont be possible.help me if you have better way.

Reply:
You can define a new interface that extends the old one, or you can use classes instead of interfaces, or you can prepare a major incompatible version change.